### PR TITLE
Remove unused columns

### PR DESCRIPTION
**Pull request by agorina**
I am not going to merge it until the rest of the system is adjusted, but would like to get feedback on this.

### CODE REVIEW

File: reporting/sql/V1_2_1_5__config_subject_cleanup.sql [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ALTER TABLE grade
+   DROP COLUMN name;
+
+ALTER TABLE asmt_type
+   DROP COLUMN name;
+
+ALTER TABLE item
+   DROP COLUMN target_code,
+   DROP COLUMN dok_level_subject_id;
+
```
[the remark this answers is not in the document]
Comment (wtritch) [reply reporting/sql/V1_2_1_5__config_subject_cleanup.sql]: I think that's all of them.

File: warehouse/sql/V1_2_1_4__config_subject_cleanup.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
+-- Remove unused columns
+use ${schemaName};
+
+ALTER TABLE depth_of_knowledge
+    DROP COLUMN description;
```
Comment (malaffoon) [inline warehouse/sql/V1_2_1_4__config_subject_cleanup.sql]: These aren't used but they do help when checking the references. Does the configurable subject not allow for these at all?

File: warehouse/sql/V1_2_1_4__config_subject_cleanup.sql [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+ALTER TABLE claim
+    DROP COLUMN name,
+    DROP COLUMN description;
+
+ALTER TABLE common_core_standard
+    DROP COLUMN description;
```
[the remark this answers is not in the document]
Comment (agorina) [reply warehouse/sql/V1_2_1_4__config_subject_cleanup.sql]: `subject_translation`, and it gets loaded as part of the subject's xml.

File: warehouse/sql/V1_2_1_4__config_subject_cleanup.sql [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+ALTER TABLE subject_claim_score
+    DROP COLUMN name;
+
+ALTER TABLE target
+    DROP COLUMN code,
+    DROP COLUMN description,
+    MODIFY COLUMN natural_id varchar(20) not null;
+
+ALTER TABLE grade
+    DROP COLUMN name;
```
Comment (malaffoon) [inline warehouse/sql/V1_2_1_4__config_subject_cleanup.sql]: Again, these aren't used but, quick, what is `IT` again?

**Pull request merged**